### PR TITLE
APPOPS-17333 Migrated to v6, changed to vite

### PR DESCRIPTION
**Pull request by jakub-jarek**

### CODE REVIEW

**Comment by jakub-jarek** [comment]
2 things left behind:

https://app.layer0.co/signup because i dont know any other

edgio-docs-edgio-ecommmerce-api-example-default.edgio-limelight.link API doesnt work for me, used the old layer0-docs-layer0-ecommmerce-api-example-default.layer0-limelight.link/

File: README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
-# Deploy Vue 3 example to Layer0
+# Deploy Vue 3 example to Edgio
 
-A demo deployment of Vue 3 app to Layer0.
+A demo deployment of Vue 3 app to Edgio.
 
 ## Demo
 
 https://layer0-docs-layer0-vue3-example-default.layer0-limelight.link
```
Comment (jakub-jarek) [inline README.md:7]: i expect it to be deployed under old url

File: edgio.config.js [hunk 2]
```diff
@@ -1,24 +1,59 @@
...
+  // The name of the team in Edgio to which this app should be deployed.
+  // team: 'my-team-name',
+
   backends: {
     // Define a domain or IP address to proxy as a backend
-    // More on: https://docs.edgio.co/guides/edgio_config#backends
+    // More on: https://docs.edg.io/guides/layer0_config#backends
     api: {
-      domainOrIp: 'edgio-docs-edgio-ecommmerce-api-example-default.edgio-limelight.link',
-      hostHeader: 'edgio-docs-edgio-ecommmerce-api-example-default.edgio-limelight.link',
```
Comment (jakub-jarek) [inline edgio.config.js:12]: this was probably caused by global string replace, as the url is invalid -> deployments fail, so no new version is deployed, therefore it looks like it works

File: .github/workflows/layer0.yml [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
         with:
-          node-version: 14
+          node-version: 16
           registry-url: https://npm-proxy.fury.io/moovweb/
       - run: npm ci
-      - run: npm run layer0:deploy -- --token=$LAYER0_DEPLOY_TOKEN
+      - run: npm run edgio:deploy -- --token=$EDGIO_DEPLOY_TOKEN
         env:
-          LAYER0_DEPLOY_TOKEN: ${{secrets.LAYER0_DEPLOY_TOKEN}}
+          EDGIO_DEPLOY_TOKEN: ${{secrets.EDGIO_DEPLOY_TOKEN}}
```
Comment (rishi-raj-jain) [inline .github/workflows/layer0.yml]: ${{secrets.LAYER0_DEPLOY_TOKEN}} can be used for now, tho ideally we shal l change to EDGIO_DEPLOY_TOKEN

File: .gitignore [hunk 4]
```diff
@@ -22,6 +22,16 @@ pnpm-debug.log*
 *.sln
 *.sw?
 
-# Layer0 generated build directory
-.layer0
+# Edgio generated build directory
+.edgio
+.edgio_temp
```
Comment (rishi-raj-jain) [inline .gitignore:28]: Is temp directory generated as of now?

File: README.md (hunk 1 above)
Comment (rishi-raj-jain) [reply README.md:7]: Yup, I think @tristanlee85 is in process of migrating to other repo

File: .gitignore (hunk 4 above)
Comment (jakub-jarek) [reply .gitignore:28]: yes

File: edgio.config.js (hunk 2 above)
Comment (rishi-raj-jain) [reply edgio.config.js:12]: Thank you, yes!

File: index.html [hunk 5]
```diff
@@ -1,24 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
     <meta property="og:locale" content="en_US" />
     <meta property="og:type" content="website" />
     <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+    <!-- TODO: twitter:card? -->
```
Comment (rishi-raj-jain) [inline index.html]: isn't it just below that?

File: src/main.js [hunk 6]
```diff
@@ -1,15 +1,25 @@
...
 installDevtools()
+
+new Metrics({
+  token: '789a5742-3d8d-4894-90ec-6ca44b91f914',
+  router: new Router()
+    .match('/', ({ setPageLabel }) => setPageLabel('home'))
+    .match('/commerce', ({ setPageLabel }) => setPageLabel('commerce'))
+    .match('/product/:id', ({ setPageLabel }) => setPageLabel('product/:id')),
+}).collect()
+
```
Comment (rishi-raj-jain) [inline src/main.js:24]: Can we break this into components?

File: src/views/CommerceView.vue [hunk 7]
```diff
@@ -61,18 +63,13 @@ export default {
...
   },
   watch: {
-    '$route.query.filter': {
-      handler: function (filter) {
-        this.createFilter()
-        this.setProducts(this.$route.params.slug, filter)
-      },
-      deep: true,
-      immediate: true,
-    },
```
Comment (rishi-raj-jain) [inline src/views/CommerceView.vue:71]: Why remove the watcher?

Comment (jakub-jarek) [reply src/views/CommerceView.vue:71]: because it was causing flicker, doing 3 requests instead of 1

File: index.html (hunk 5 above)
Comment (jakub-jarek) [reply index.html]: removed

File: .github/workflows/layer0.yml (hunk 3 above)
Comment (jakub-jarek) [reply .github/workflows/layer0.yml]: changed

File: README.md (hunk 1 above)
Comment (jakub-jarek) [reply README.md:7]: ok

File: src/views/CommerceView.vue (hunk 7 above)
Comment (rishi-raj-jain) [reply src/views/CommerceView.vue:71]: Ah okay, but now if I select a sort by on the sidebar, does the UI change acc. to that?

Comment (jakub-jarek) [reply src/views/CommerceView.vue:71]: yes, everything should be ok -> https://jakub-jarek-edgio-vue3-example-default.layer0-limelight.link

Comment (rishi-raj-jain) [reply src/views/CommerceView.vue:71]: yup, and fast!

File: src/main.js (hunk 6 above)
Comment (rishi-raj-jain) [reply src/main.js:24]: Ignore it

**Pull request closed without merging**